app/src/main/java/com/vultisig/wallet/ui/screens/swap/SwapScreen.kt

```diff
@@ -429,12 +429,6 @@ internal fun SwapScreen(
                                     onSelectSrcPercentage(0.75f)
                                 },
                             )
-                            PercentageItem(
-                                title = "MAX",
-                                onClick = {
-                                    onSelectSrcPercentage(1f)
-                                },
-                            )
```

Comment:
we should only remove for native coins @aminsato . It is completely fine to have it for tokens

Reply:
Let's keep it consistent , currently IOS / windows all removed the max button